PusherMainView/PusherMainView/Base.lproj/Pusher.storyboard

```diff
@@ -128,7 +128,7 @@
                                             <rect key="frame" x="0.0" y="0.0" width="460" height="144"/>
                                             <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
                                             <subviews>
-                                                <textView importsGraphics="NO" richText="NO" verticallyResizable="YES" spellingCorrection="YES" smartInsertDelete="YES" id="OgG-Ak-can">
+                                                <textView importsGraphics="NO" richText="NO" verticallyResizable="YES" allowsUndo="YES" spellingCorrection="YES" smartInsertDelete="YES" id="OgG-Ak-can">
```

Comment:
Lol. Easier than I thought.
Does it mean redo will work as well?

Reply:
Yes redo works too 👍 @imaginaris